src/Symfony/Component/ExpressionLanguage/Tests/LexerTest.php

```diff
@@ -35,6 +35,32 @@ public function testTokenize($tokens, $expression)
         $this->assertEquals(new TokenStream($tokens, $expression), $this->lexer->tokenize($expression));
     }
 
+    public function testTokenizeMultilineComment()
+    {
+        $expression = <<<EXPRESSION
+            /**
+             * This is 2^16, even if we could have
+             * used the hexadecimal representation 0x10000. Just a
+             * matter of taste! 🙂
+             */
+            65536 and
+            /*
+             * This time, only one star to start the comment.
+             * The previous one was valid too!
```

Comment:
I suggets removing the leading `*` on those lines, to avoid implying to readers of the test that they are necessary for multiline comments.

Reply:
Right, updated 👍